awsmp/models.py

```diff
@@ -648,6 +649,37 @@ def get_entity_from_yaml(yaml_config: dict[str, Any]) -> EntityModel:
                 "Regions": ami_product.region.commercial_regions,
                 "FutureRegionSupport": ami_product.region.future_region_supported()[-1],
             },
+            "Versions": {
```

Comment:
We call this "Versions", but only support a single option. Would it make more sense to have this be a list of dictionaries so we _could_ show multiple versions, even if we don't strictly pull them on download

Reply:
Jess pointed out that version is also used for version update/initial version setting by awsmp so this would be a breaking change. Definitely out of scope for this PR, but we should revisit this topic soon :tm: